### PR TITLE
Update allow-private-services-pods.yaml

### PR DESCRIPTION
**Pull request by IdanAdar**
Add IP addresses for Sysdig and Logging in eu-es

### CODE REVIEW

**Review by derekpoindexter — APPROVED** [review]

File: calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml [hunk 1]
```diff
@@ -33,6 +33,9 @@ spec:
       - 166.9.28.71/32
       - 166.9.30.53/32
       - 166.9.32.51/32
+      - 166.9.94.31/32
```
Comment (bradbehle) [inline calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml:36]: Are these going to be added to https://cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints ?

File: calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml [hunk 2]
```diff
@@ -52,7 +55,10 @@ spec:
       - 166.9.30.45/32 
       - 166.9.28.159/32 
       - 166.9.32.117/32
-      # End new IP address list
+      # IPs for eu-es
```
Comment (bradbehle) [inline calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml:58]: Are these going to be added to https://cloud.ibm.com/docs/log-analysis?topic=log-analysis-endpoints ?

File: calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml (hunk 1 above)
Comment (bradbehle) [reply calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml:36]: Also, these, and the IPs below should also be added to https://github.com/IBM-Cloud/kube-samples/blob/master/calico-policies/private-network-isolation/eu-es/allow-private-services.yaml

**Review by bradbehle — CHANGES_REQUESTED** [review]

File: calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml (hunk 2 above)
Comment (IdanAdar) [reply calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml:58]: they're there in test.cloud

File: calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml (hunk 1 above)
Comment (IdanAdar) [reply calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml:36]: yes they're from test.cloud

Comment (bradbehle) [reply calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml:36]: https://test.cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints lists

166.9.36.71
166.9.34.56 

as the eu-es endpoints

Comment (bradbehle) [reply calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml:36]: Confirmed that the IPs in the docs are incorrect, and the ones in this PR are correct

**Review by bradbehle — APPROVED** [review]

**Pull request merged**